mozilla_django_oidc/views.py

```diff
@@ -114,15 +114,15 @@ def redirect_url(self):
 
     def dispatch(self, request, *args, **kwargs):
         """Log out the user."""
-
-        if request.user.is_authenticated():
+        # Check if a method exists to build the URL to log out the user
+        # from the OP.
+        logout_from_op = import_from_settings('OIDC_OP_LOGOUT_URL_METHOD', '')
```

Comment:
I agree--if the user isn't logged in to the app, then this view should essentially be a no-op.